dashboard/app/models/user_level.rb

```diff
@@ -32,7 +32,8 @@ class UserLevel < ActiveRecord::Base
   belongs_to :script
   belongs_to :level_source
 
-  before_save :handle_unsubmit, if: ->(ul) {ul.submitted_changed?(from: true, to: false)}
+  after_save :after_submit, if: ->(ul) {ul.submitted_changed? to: true}
+  before_save :before_unsubmit, if: ->(ul) {ul.submitted_changed? from: true, to: false}
```

Comment:
Considered reducing this condition to
```suggestion
  before_save :before_unsubmit, if: ->(ul) {ul.submitted_changed? from: true}
```
but wasn't sure if we think that's risky at this point.  Thoughts?

Reply:
This sounds ok to me, but I don't have context to know whether you can 'unsubmit' and end up with a `nil` submitted status